packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/tspconfig.yaml

```diff
@@ -1,6 +1,7 @@
 options:
-  "@azure-tools/typespec-csharp":
+  "@typespec/http-client-csharp":
```

Comment:
lets remove `generate-convenience-methods`, `head-as-boolean`, `generate-test-project`, `deserialize-null-collection-as-null-value` since none of these parameters are implemented and most likely will not be implemented in the same way.

If we want to test pass through of tspconfig.yaml params to an extension emitter we should do that in unit tests

Reply:
OK. Then we will not config the Unbranded-TypeSpec, and I remove the tspconfig.yaml file.